owtf/webapp/src/components/ConfigurationTabsContent/index.js

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
@sharmamohit123 Can you make a folder in components named, SettingsPage and put  ConfigurationTabsContent.js in that. That way it will be more organized.